code/src/Components/SkillsQuestion.js

```diff
@@ -0,0 +1,129 @@
+                            onChange={()=>onSkillsArrayChange('HTML')}/>
+                            <span className="checkmarkCheck"></span>
+                        </label>
+                    </div>
+                    <div className="skillContainer">
+                        <label className="checkBtnContainer">JavaScript
+                            <input 
+                            type="checkbox" 
+                            value='JavaScript'
+                            checked={skillsArray.includes('JavaScript')}
+                            onChange={()=>onSkillsArrayChange('JavaScript')}/>
+                            <span className="checkmarkCheck"></span>
+                        </label>
+                    </div>
+                    <div className="skillContainer">
+                        <label className="checkBtnContainer">Node.js
+                            <input 
+                            type="checkbox" 
+                            value='Node.js'
+                            checked={skillsArray.includes('Node.js')}
+                            onChange={()=>onSkillsArrayChange('Node.js')}/>
+                            <span className="checkmarkCheck"></span>
+                        </label>
+                    </div>
+                    <div className="skillContainer">
+                        <label className="checkBtnContainer">React
+                            <input 
+                            type="checkbox" 
+                            value='React'
+                            checked={skillsArray.includes('React')}
+                            onChange={()=>onSkillsArrayChange('React')}/>
+                            <span className="checkmarkCheck"></span>
+                        </label>
+                    </div>
+                    <div className="skillContainer">
+                        <label className="checkBtnContainer">Redux
+                            <input 
+                            type="checkbox" 
+                            value='Redux'
+                            checked={skillsArray.includes('Redux')}
+                            onChange={()=>onSkillsArrayChange('Redux')}/>
+                            <span className="checkmarkCheck"></span>
+                        </label>
+                    </div>
+                    
+                </div>
+
+                <div className="buttons">
+                <button className="button"
+                    onClick={onStepChange}
+                    >Next <span className="arrow right"></span></button>
+                <button className="button"
+                    onClick={onStepBackChange}
+                    ><span className="arrow left"></span> Back</button>
+                
+            </div>
+            <div className="progress-container">
+                <div style={{width: `${progress}%`}} className="progress-value">
+                </div>
+            </div>
+        </div>
+      )
```

Comment:
this is also a very long component, if possible it would be better to find a way to make it shorter so you easier could make changes and more bug proof :)